controllers/drcluster_controller.go

```diff
@@ -936,51 +936,9 @@ func (u *drclusterInstance) removeFencingCR(cluster ramen.DRCluster) (bool, erro
 	err := u.mwUtil.DeleteManifestWork(fmt.Sprintf(util.ManifestWorkNameFormat,
 		u.object.Name, cluster.Name, util.MWTypeNF), cluster.Name)
 	if err != nil {
-		if errors.IsNotFound(err) {
```

Comment:
So, why are you returning an error if the manifestwork was deleted and does not exist?

Reply:
I return error if DeleteManifestWork function failed.